lessons/11-basic-data-structs/assignments/account-generator.md

```diff
@@ -0,0 +1,28 @@
+# Student Account Generator
+
+We are going to write a program that generates some student information. This exercise will practice our skills with strings, random numbers, arrays and iterators.
+
+## Requirements
+
+1. You will start by creating three arrays:
+    - An array that will contain student names
+    - An array that will store student ID numbers
+    - An array that will contain student email addresses    
+
+2. Write a `times` loop that asks the user to enter in 5 student names (first and last name) and store those values in the first array in **all uppercase**.
```

Comment:
maybe add: Example result is `students = ["ROSIE MARTINEZ", "JOE LIU", "SALLY SUE", BOB JOHNSON", "DELILA AGHO"]`